server/cherry_pick_test.go

```diff
@@ -81,6 +81,17 @@ func TestHandleCherryPick(t *testing.T) {
 		err = s.handleCherryPick(context.Background(), "org-member", "/cherry-pick release-5.28", pr)
 		require.EqualError(t, err, "too many requests")
 	})
+
+	t.Run("should not panic on empty requests", func(t *testing.T) {
+		defer func() {
+			if r := recover(); r != nil {
+				require.Failf(t, "recovered from panic", "%v", r)
+			}
+		}()
+
+		err := s.handleCherryPick(context.Background(), "org-member", "/cherry-pick", pr)
+		require.NoError(t, err)
+	})
```

Comment:
Any reason for not using `require.NotPanicsf()` here?

Reply:
Oh, didn't even realize there was a method for that!

`assert.NotPanics(t, func(){ RemainCalm() })` - love this :D